cluster-operator/src/test/java/io/strimzi/operator/cluster/model/SecretGeneratorTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.fabric8.kubernetes.api.model.OwnerReference;
+import io.fabric8.kubernetes.api.model.OwnerReferenceBuilder;
+import io.fabric8.kubernetes.api.model.Secret;
+import io.strimzi.operator.common.model.Labels;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+
+class SecretGeneratorTest {
+
+    final static String NAME = "secret-name";
+    final static String NAMESPACE = "my-namespace";
+    final static Labels LABELS = Labels.EMPTY.withStrimziName("component");
```

Comment:
You do not need to use `static` keyword or do you also using these instance variables inside @AfterAll or @BeforeAll phases?